t/phase.t

```diff
@@ -0,0 +1,273 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
```

Comment:
(also if you have any idea of why Travis was/is failing installing Test::Nginx via cpanm, lemme know)